variables.tf

```diff
@@ -113,11 +113,64 @@ variable "log_group_name" {
   default = ""
 }
 
+variable "system_log_group_name" {
```

Comment:
all this new added variables are for fluent-bit, right?
better to have them in one object variable named for example fluent_bit, somethig like this
variable fluent_bit {
type = object({
  system_log_group_name = string
  values_yaml = string
  ....
})
}